Assets/StreamingAssets/Quests/A0C01Y01.txt

```diff
@@ -119,7 +119,7 @@ _S.00_ task:
 	end quest 
 
 _S.01_ task:
-	daily from 21:00 to 22:59
+	daily from 21:00 to 23:59
```

Comment:
When you change this from 23:59 to 22:59, you added the `_latehour_` task below.
https://github.com/Interkarma/daggerfall-unity/commit/21fadbd6cbf32888bb070fcbe786bd5e4f87f8db

Is the `_latehour_` still needed or not?

Reply:
Good eye. I added latehour to ensure the enemy would have to spawn somehow, and that happens at 11PM. I restored the original spawning window and kept latehour as it is.